src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -136,6 +139,9 @@ export class QuestionSubmissionFormComponent implements OnInit {
    * Triggers the change of the recipient submission form.
    */
   triggerRecipientSubmissionFormChange(index: number, field: string, data: any): void {
+    this.isQuestionValid.subscribe((isValid: boolean) => {
+      this.model.isValid = isValid;
```

Comment:
You should use `formModelChange` to fire event rather than change directly here.

Reply:
Ideally since the model is being emitted in this method, I'd want to ensure that its `isValid` attribute is up to date

Reply:
It's not about safe or not. It's violating the principle we want to enforce: child component should communicate with parent component by emitting events.